packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -139,4 +157,23 @@ function CustomViewShell(props: TCustomViewShellProps) {
   );
 }
 
-export default CustomViewShell;
+const CustomViewShellWrapper = (props: TCustomViewShellProps) => {
+  if (process.env.NODE_ENV === 'development' && !props.disableDevHost) {
+    return (
+      <Suspense fallback={<ApplicationLoader />}>
+        <CustomViewDevHost applicationMessages={props.applicationMessages}>
+          <CustomViewShell applicationMessages={props.applicationMessages}>
+            {props.children}
+          </CustomViewShell>
+        </CustomViewDevHost>
+      </Suspense>
+    );
+  }
```

Comment:
For local development we're wrapping the `<CustomViewShell>` with `<CustomViewDevHost>`